py/picca/tests/test_2_pk1d.py

```diff
@@ -69,7 +69,7 @@ def test_Pk1D_raw(self):
         ### Test
         if self._test:
             path1 = self._masterFiles + "/test_Pk1D/Pk1D_raw.fits.gz"
-            path2 = self._branchFiles + "/Products/Pk1D_raw/Pk1D-0.fits.gz"
+            path2 = self._branchFiles + "/Products/Pk1D_raw/Pk1D-272.fits.gz"
```

Comment:
those numbers are now propagated from the delta files, right? So they are deterministic?

Reply:
yes, I choose to save the healpix number instead of the index number which does not make sense. It will be usefull for P3D stuff